### PR TITLE
Update collection-index.yml with m4tchl0ck Features 

### PR DESCRIPTION
**Pull request by m4tchl0ck**
## What type of PR is this?

- [x] Add a new dev container collection
- [ ] Update to an existing dev container collection
- [ ] Documentation/spec update
- [ ] Other containers.dev site update (UX, layout, etc)

## Related Issues

- Related Issue #
- Closes #

## Description

_Please replace this line with a description of your PR._

### Collection checklist
_If your PR contributes a new collection, please utilize this checklist:_
- [x] Collection name
- [x] Maintainer name
- [x] Maintainer contact link (i.e. link to a GitHub repo, email)
- [x] Repository URL
- [x] OCI Reference
- [x] I acknowledge that this collection provides new functionality, distinct from the existing collections part of this index.

### CODE REVIEW

File: _data/collection-index.yml [hunk 1]
```diff
@@ -773,3 +773,8 @@
   contact: https://github.com/skriptfabrik/devcontainer-features/issues
   repository: https://github.com/skriptfabrik/devcontainer-features/
   ociReference: ghcr.io/skriptfabrik/devcontainer-features
+- name: Dev Container Features by m4tchl0ck
+  maintainer: Adrian Rusznica
+  contact: https://github.com/m4tchl0ck/devcontainer-features/issues
```
Comment (samruddhikhandale) [inline _data/collection-index.yml:778]: Looks like the host repository is private?

Can you help change the visibility of this repo and "metadata" package (and your Feature packages) to public? The [index](https://containers.dev/features) won't be able to crawl the Features in `m4tchl0ck/devcontainer-features` if it's private.

Feel free to look at https://containers.dev/guide/author-a-feature#publishing for information.

**Review by samruddhikhandale — CHANGES_REQUESTED** [review]

File: _data/collection-index.yml (hunk 1 above)
Comment (m4tchl0ck) [reply _data/collection-index.yml:778]: It is strange because I see it packages public
image
and repository also
image

Comment (samruddhikhandale) [reply _data/collection-index.yml:778]:
```suggestion
  contact: https://github.com/m4tchl0ck/devcontainers-features/issues
```

Looks like there are typos in this new entry 😅

Comment (m4tchl0ck) [reply _data/collection-index.yml:778]: :) thank you

**Comment by m4tchl0ck** [comment]
I will change the repository name and will came back :)

**Comment by samruddhikhandale** [comment]
> I will change the repository name and will came back :)

@m4tchl0ck FYI - If it's your preference thing, then changing repo name is fine. However, we don't require you to have a specific name, it can be anything you'd like.

**Comment by m4tchl0ck** [comment]
I fixed the branch name :)

**Review by samruddhikhandale — APPROVED** [review]
Looks good, thank you! 🎉

**Pull request merged**